nextstrain_profiles/nextstrain-gisaid-21L/builds.yaml

```diff
@@ -0,0 +1,648 @@
+  oceania_all-time:
+    subsampling_scheme: nextstrain_region_grouped_by_division_all_time
+    region: Oceania
+    title: Evolution SARS-CoV-2 relative to clade 21L reference virus with subsampling focused on Oceania since pandemic start
+  south-america_2m:
+    subsampling_scheme: nextstrain_region_grouped_by_country_2m
+    region: South America
+    title: Evolution SARS-CoV-2 relative to clade 21L reference virus with subsampling focused on South America over the past 2 months
+  south-america_6m:
+    subsampling_scheme: nextstrain_region_grouped_by_country_6m
+    region: South America
+    title: Evolution SARS-CoV-2 relative to clade 21L reference virus with subsampling focused on South America over the past 6 months
+  south-america_all-time:
+    subsampling_scheme: nextstrain_region_grouped_by_country_all_time
+    region: South America
+    title: Evolution SARS-CoV-2 relative to clade 21L reference virus with subsampling focused on South America since pandemic start
+
+# remove sequences without division label in US and sequences from prior to clade 21L
+filter:
+  min_date: "2022-01-01"
+  exclude_where: "division='USA' \
+    Nextstrain_clade='19A' \
+    Nextstrain_clade='19B' \
+    Nextstrain_clade='20A' \
+    Nextstrain_clade='20B' \
+    Nextstrain_clade='20C' \
+    Nextstrain_clade='20D' \
+    Nextstrain_clade='20E (EU1)' \
+    Nextstrain_clade='20F' \
+    Nextstrain_clade='20G' \
+    Nextstrain_clade='20H (Beta, V2)' \
+    Nextstrain_clade='20I (Alpha, V1)' \
+    Nextstrain_clade='20J (Gamma, V3)' \
+    Nextstrain_clade='21A (Delta)' \
+    Nextstrain_clade='21B (Kappa)' \
+    Nextstrain_clade='21C (Epsilon)' \
+    Nextstrain_clade='21D (Eta)' \
+    Nextstrain_clade='21E (Theta)' \
+    Nextstrain_clade='21F (Iota)' \
+    Nextstrain_clade='21G (Lambda)' \
+    Nextstrain_clade='21H (Mu)' \
+    Nextstrain_clade='21I (Delta)' \
+    Nextstrain_clade='21J (Delta)' \
+    Nextstrain_clade='21K (Omicron)' \
+    Nextstrain_clade='21M (Omicron)'"
+
+subsampling:
+
+  # Custom subsampling logic for group by clade
+  nextstrain_reference:
+    clades:
+      group_by: "Nextstrain_clade"
+      max_sequences: 300
+
+  # Custom subsampling logic for regions over 2m
+  # Grouping by division for North America and Oceania
+  # 4000 total
+  # 4:1 ratio of recent to early
+  # 4:1 ratio of focal to context
+  nextstrain_region_grouped_by_division_2m:
+    # Early focal samples for region
+    focal_early:
+      group_by: "division year month"
+      max_sequences: 640
+      max_date: "--max-date 2M"
+      exclude: "--exclude-where 'region!={region}'"
+    # Early contextual samples from the rest of the world
+    context_early:
+      group_by: "country year month"
+      max_sequences: 160
+      max_date: "--max-date 2M"
+      exclude: "--exclude-where 'region={region}'"
+    # Recent focal samples for region
+    focal_recent:
+      group_by: "division year month"
```

Comment:
This should switch to `division week` (and others `country week`) like master, along with similar subsampling schemes and samples.

Again, not obvious here, but obvious in [a copy + change diff](https://tsibley.net/paste/2023-03-30Nk3QbSco-tmp-28232-LkyTxc.html).